models/DataObject/AbstractObject.php

```diff
@@ -995,6 +987,7 @@ public function setParentId($o_parentId)
         }
         $this->o_parentId = $o_parentId;
         $this->o_parent = null;
+        $this->o_siblings = null;
```

Comment:
```
$this->o_siblings = [];
$this->o_hasSiblings = [];
```

Reply:
@dvesh3 resetting on `setParentId()` should be also considered for documents.

Reply:
done! thnx :)